src/platform_projects/openthread-efr32-rcp-spi.slcp

```diff
@@ -0,0 +1,23 @@
+project_name: openthread-efr32-rcp-spi
+label: ot-efr32
+package: OpenThread
+description: This is the platform project that will enable the ot-efr32 repo on GitHub to generate a CMake library for EFR32 RCPs which communicate over SPI.
+category: OpenThread Examples
+quality: production
+
+component:
+  - id: brd4001a
```

Comment:
We don't need/want the brd4001a. This was removed from all the slcp files in the GSDK.  Probably need to fix the uart version as well.

Reply:
Yes, cleaned up from all the .slcps.